pkgs/by-name/gd/gdscript-formatter/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "gdscript-formatter";
+  version = "0.9.1";
```

Comment:
`0.10.0` is out and it should fix the hash mismatch since it now uses the upstream `tree-sitter-gdscript`, which doesn't have the directory conflict.